src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -25,11 +25,12 @@ import {log} from 'shared/logging/config';
 import {DeviceEventEmitter, Platform} from 'react-native';
 import {ContagiousDateInfo, ContagiousDateType} from 'shared/DataSharing';
 import {EN_API_VERSION, QR_ENABLED} from 'env';
-import {EventTypeMetric, FilteredMetricsService} from 'services/MetricsService/FilteredMetricsService';
+import {FilteredMetricsService} from 'services/MetricsService/FilteredMetricsService';
 import {checkNotifications} from 'react-native-permissions';
 import {Status} from 'screens/home/components/NotificationPermissionStatus';
 import {PollNotifications} from 'services/PollNotificationService';
 import {OutbreakService} from 'shared/OutbreakProvider';
+import {EventTypeMetric} from 'services/MetricsService/MetricsFilter';
```

Comment:
Pending comment above this would be moved up and into a single import

Reply:
Done